tests/suites/test_suite_cipher.function

```diff
@@ -712,3 +712,75 @@ void check_padding( int pad_mode, char *input_str, int ret, int dlen_check )
         TEST_ASSERT( dlen == (size_t) dlen_check );
 }
 /* END_CASE */
+
+/* BEGIN_CASE depends_on:MBEDTLS_CIPHER_MODE_GCM */
+void check_gcm( int cipher_id, char *hex_key, char *hex_iv )
```

Comment:
I believe this is a test for special use case of short buffers. Please rename the function name

Reply:
This function encrypts a 1024 bytes buffer in one time and then encrypts the same 1024 bytes buffer piece by piece.  So the test is not specially to short buffers. This test is here to show that we can encrypt large buffer piece by piece but not only 16 bytes by 16 bytes.